test/.env

```diff
@@ -11,7 +11,7 @@ PIXL_DB_PORT=5432
 PIXL_DB_NAME=pixl
 PIXL_DB_USER=pixl_db_username
 PIXL_DB_PASSWORD=pixl_db_password
-SKIP_ALEMBIC=false
+SKIP_ALEMBIC=true
```

Comment:
the system tests shouldn't skip alembic. We want this in there so we can know that we've added our migrations to source control and it'll run in production

Reply:
SKIP_ALEMBIC=false : [eefc515](https://github.com/UCLH-Foundry/PIXL/pull/484/commits/eefc515dd69a08339b5958b0702ea8c992d6ece1)